### ISSUE TITLE
Classify output of joystick X axis 🕹️

### ISSUE THREAD

**Issue by slevin48** [slevin48/marioKartAI#2]
Change last layer of the Neural Network from regression to classification, by specifying the number of classes (OUT_SHAPE).
Change the loss function to Cross Entropy, either:
1. Classique: https://www.tensorflow.org/api_docs/python/tf/keras/losses/CategoricalCrossentropy
or
2. Sparse: https://www.tensorflow.org/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy

Depending on one-hot encoded (1) or ordinal (2):
https://stats.stackexchange.com/questions/326065/cross-entropy-vs-sparse-cross-entropy-when-to-use-one-over-the-other
